ui/lib/kubernetes/addon/components/page/role/create-and-edit.hbs

```diff
@@ -116,10 +116,14 @@
             @valueUpdated={{fn (mut template.rules)}}
             @helpText={{sanitized-html this.roleRulesHelpText}}
           >
-            <button type="button" class="toolbar-link" {{on "click" this.resetRoleRules}} data-test-restore-example>
-              Restore example
-              <Icon @name="reload" />
-            </button>
+            <Hds::Button
+              @icon="reload"
+              @text="Restore example"
+              @color="secondary"
+              class="toolbar-link"
```

Comment:
`toolbar-link` might need to be changed to `toolbar-button` (that's what I had to do for nav toolbars, I think JsonEditor uses the same toolbar component)

Reply:
Good call! I merged in the changes from your last PR and I will update the codemod to change the class for future runs.